server-install/server-install.sh

```diff
@@ -45,52 +47,3 @@ sudo snap start docker
 sleep 5
 # This is needed to get snap Docker to initialize fully
 docker run hello-world
-
-# Install Pelias
```

Comment:
If `./install-pelias` needs to be executed separately, please add instructions somewhere do do so.

Reply:
Good call. READMEs updated in 25f168a42ca47bafcdaeffd5ca322b80effc89b7